src/style.css

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
use * selector only for box-sizing property. Don't use it for all the other properties